hnn_core/viz.py

```diff
@@ -82,6 +96,15 @@ def plot_dipole(dpl, tmin=None, tmax=None, ax=None, layer='agg', decim=None,
         The SciPy function :func:`~scipy.signal.decimate` is used, which
         recommends values <13. To achieve higher decimation factors, a list of
         ints can be provided. These are applied successively.
+    units : str | None
+        The physical units of the data, used for axis label. Defaults to
+        ``units='nAm'``. Passing ``None`` results in units being omitted from
+        plot.
+    scaling : float | None
+        The scaling to apply to the dipole data in order to achieve the
+        specified ``units`` when plotting. Defaults to None, which applies
+        unit scaling (1x). For example, use``scaling=1e-6`` to scale fAm to
+        nAm.
```

Comment:
why is there a `None` option? It makes sense if `scaling` could be a `dict` or `list`. Because have an empty list or dict as default is dangerous